### DISCUSSION TITLE
Call and SMS via SEPIA

### DISCUSSION THREAD

**Discussion by giostark** [SEPIA-Framework/sepia-docs#158] [General]
Hi,
I read this post : https://github.com/SEPIA-Framework/sepia-docs/issues/110 about the SMS function.
Would be REALLY  great if it will be possible make call and sending sms from SEPIA.
I know this is an hard to implement function but I write here just to point how useful those would be. (and make this topic hight)
Many of my friends (just users) would benefit of those features.
1) use alarms.
2) navigation.
3) sms.
4) call.
5) make search.
6) etc etc ...

Lots of those functions are already here.
Call and sms could be even business functions of everyday , that would make this project even more hugely wide-spread. 
Also the old parents that are not skilled with phone (and need thick glasses to watch it) could benefit of simple commands as "Call -person- " or "text -person - content ".  
Sometimes I'm conflicted that make the things easier for the most will feed laziness (for don't use other expression)  , but sometimes those are really necessary or extremely useful.

**Comment by fquirin** [comment]
It is an interesting feature but I mentioned a few concerns about it in #110 . First of all it is a feature that natively only works on Android (and probably iOS), every other client would require a third party SMS service and phone calls might not work at all. The next thing is to find the correct phone number via some contacts management.
That said I think it's currently possible to "start" a phone call if you use the 'Open link/website' Teach-UI  command with URLs like `tel:123-456`. It should open the phone app with the correct number, you still have to press the phone button though.
